src/describe.cpp

```diff
@@ -72,5 +89,26 @@ int main(int argc, char** argv)
                                         << counts[3] << ")\n";
         std::cout << oss.str();
     }
+
+    if (argc > 3) {
+        std::string name = std::string(argv[3]);
+        int dim = atoi(argv[4]);
+        int value = atoi(argv[5]);
+        auto tagbase = mesh.get_tagbase(dim, name);
+        int numEq = 0;
+        if (tagbase->type() == OMEGA_H_I8)
+            numEq = getNumEq<Omega_h::I8>(mesh, name, dim, value);
```

Comment:
I think it would be more useful for the tool to count the 'owned' mesh entities (via https://github.com/SCOREC/omega_h/blob/ecee2854085d6e3f7d6c79064efaaea2dfaa539c/src/Omega_h_mesh.hpp#L322) that have the value and return/print the sum of that result across all ranks.  For meshes with dozens of parts I think the per-rank output may be less useful as a debugging/sanity check.

Reply:
We'll see how folks use the tool and then decide if this is needed.